<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuestionControl</class>
 <widget class="QDialog" name="QuestionControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="subjectList">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>761</width>
     <height>391</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="verticalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>176</number>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>30</number>
   </attribute>
   <column>
    <property name="text">
     <string>Тип</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Вопрос</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Предмет(ы)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Сложность</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string> </string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="addQuestion">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delQuestion">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>761</width>
     <height>381</height>
    </rect>
   </property>
   <property name="title">
    <string>Добавление</string>
   </property>
   <widget class="QComboBox" name="questionType">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Выбор вариантов</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ввод ответа</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Последовательность</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Группировка</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="finishQustionAdd">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить к текущему предмету</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Тип вопроса</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Кол-во вариантов</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="answersNumber">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="questionEdit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>491</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>90</y>
     <width>261</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить для текущего предмета</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>60</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Сформировать экзамен с выбранными вопросами</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addToSubject">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>120</y>
     <width>261</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить к предмету:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="subjectList2">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>120</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Предмет, к которому требуется добавить вопрос</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
